research/src/pages/design-systems.mdx

```diff
@@ -0,0 +1,13 @@
+---
+menu: Overview
+name: Design Systems
+path: /analysis/design-systems
+---
+
+import DesignSystemsList from '../components/design-systems-list'
+
+# List of analyzed design systems
+
+This table shows names and links to design systems which currently using during research phase.
```

Comment:
Can you reword to "This table shows names and links to design systems which are currently being used during the research phase."

Reply:
This isn't a table though. Maybe rephrase to a more generic statement:

> Below are the design systems which are often utilized by Open UI for research purposes

Reply:
I like your statement even better then my original text, since for me this page can have references to other design system which not fully documented/researched, but at the same time important to look up.